app/renderer/js/main.ts

```diff
@@ -93,6 +95,7 @@ class ServerManagerView {
 	tabs: ServerOrFunctionalTab[];
 	functionalTabs: AnyObject;
 	tabIndex: number;
+	presetOrgs: any[];
```

Comment:
We can create an interface for this too, if we know what it is going to contain.